tests/HttpRequestServiceTest.php

```diff
@@ -305,4 +310,22 @@ public function testNotJSONResponse()
 
         $this->httpRequestServiceClass->json();
     }
+
+    public function testSendWithRequestException()
+    {
+        $this->expectException(RequestException::class);
```

Comment:
could we also expect exception message by using `expectExceptionMessage`

Reply:
Done